m10_point_of_interest/01_introduction.rst

```diff
@@ -0,0 +1,61 @@
+Point Of Interest (POI)
+=======================
+
+.. styled-topic::
+
+  Main questions:
+      What are point of interest directives, why and how to use it.
+
+  Topics?
+      What, why and how.
+
+  Difficulty:
+      Easy
+
+  Laboriousness:
+      Around 5 minutes of reading
+
+
+Introduction to the Point of Interest directive.
+
+
+What is a point of interest?
+----------------------------
+
+Directive for creating a "point of interest" summary block.
+
+A point of interest is similar to a normal admonition ("coloured info box"), but
+the point of interest are also linked to each other with next/previous links. The links enable
+the user to quickly navigate between different points of interest.
+
+Point of interests may also be used to generate separate lecture slides
+(not directly included in the A+ content chapters). This requires a separate
+tool called :doc:`Presentation Maker </m10_point_of_interest/03_presentation_maker>`
+
+.. point-of-interest:: This is a point of interest
+
+  And here is the content. More examples in the :doc:`next chapter </m10_point_of_interest/02_examples>`
+
+From now on I will refer to point of interest as a POI.
+
+Why to use it?
+--------------
+
+#. Highlight important parts and make them easily accessible with links.
+
+#. Turn your Point of interest into presentation slides with Presentation Maker.
```

Comment:
This is perhaps to little content for a section. Consider making a brief introduction to the `Why to use it` and perhaps making this a subsection.

```suggestion
Why use points of interest?
...........................
Using points of interest in your course may bring some benefits. Firstly, the points of interest can be used to highlight important parts of your course content while linking these elements together in a logical and sequential manner. Finally, the points of interest can be exported as presentation slides with the Presentation Maker tools.
```
**Note:** the suggestion above is just an example, but I think you can write something better.